schedule/yast/textmode_installation_minimal_role/textmode_installation_minimal_role_x86_64.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name:           textmode_installation_minimal_role
+description:    >
+  Full Medium installation that covers the following cases:
+    1. Installation in textmode;
+    2. "Minimal" role is selected;
+    3. Boot to command-line mode;
+    4. Installation is validated by default set of smoke tests.
+vars:
+  VIDEOMODE: text
+  SYSTEM_ROLE: minimal
```

Comment:
I think we could drop this setting `SYSTEM_ROLE`.
Could you please add `VIDEOMODE` and `DESKTOP` to job group yaml, it will help later.

Reply:
added to  JobGroups/SLE_15/defaults.yaml